Doc/c-api/type.rst

```diff
@@ -477,10 +477,10 @@ The following functions and structs are used to create
       * :c:member:`~PyTypeObject.tp_vectorcall_offset`
         (see :ref:`PyMemberDef <pymemberdef-offsets>`)
 
-      Setting :c:data:`Py_tp_bases` or :c:data:`Py_tp_base` may be
+      Setting :c:data:`!Py_tp_bases` or :c:data:`!Py_tp_base` may be
```

Comment:
We may add targets for Py_tp_bases and Py_tp_base in future. It would be nice if these link worked. I think that it is better to left them as is for a reminder or temporary add them in the `nitpick_ignore` list.

Reply:
I reverted these changes.